src/main/java/org/truffleruby/options/CommandLineParser.java

```diff
@@ -557,8 +545,12 @@ private String grabOptionalValue() {
         return null;
     }
 
+    private CommandLineException notImplemented(String option) {
+        return new CommandLineException(String.format("the %s option is not implemented", option));
+    }
+
     public static void printHelp(PrintStream out) {
-        out.println("Usage: truffleruby [switches] [--] [programfile] [arguments]");
+        out.println("Usage: ruby [switches] [--] [programfile] [arguments]");
```

Comment:
I am talking about help and usage.
But indeed it would make sense to have `truffleruby` for the error prefix too:
```
$ bin/jruby -Z -e 3
jruby: unknown option -Z
```